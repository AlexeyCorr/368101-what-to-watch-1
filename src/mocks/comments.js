export const comments = [
  {
    id: 1,
    comment: `qwerty`,
    rating: 5,
    user: {
      id: 1,
      name: `name`,
    }
  }
];
