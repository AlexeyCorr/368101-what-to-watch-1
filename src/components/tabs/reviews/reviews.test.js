import React from 'react';
import renderer from 'react-test-renderer';

import {comments} from './../../../mocks/comments.js';
import Reviews from './reviews.jsx';

it(`renders correctly`, () => {
  const tree = renderer
    .create(<Reviews
      comments={comments}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
