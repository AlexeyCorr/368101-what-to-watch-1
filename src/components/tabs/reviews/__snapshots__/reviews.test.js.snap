// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  />
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          qwerty
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            name
          </cite>
          <time
            className="review__date"
            dateTime="Invalid Date"
          >
            Invalid Date
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        5
      </div>
    </div>
  </div>
</div>
`;
